app/scripts/lib/rpc-method-middleware/handlers/wallet-createSession/handler.ts

```diff
@@ -97,12 +180,14 @@ export async function walletCreateSessionHandler(req, res, _next, end, hooks) {
     const supportedEthAccounts = getEthAccounts({
       requiredScopes: supportedRequiredScopes,
       optionalScopes: supportedOptionalScopes,
+      isMultichainOrigin: true,
```

Comment:
this looks like a functional change?

Reply:
I mean it looks right but wondering why it wasn't causing issues previously?

Reply:
yeah let's do that. I'll update core

Reply:
https://github.com/MetaMask/core/pull/4835